identifiers/country-in/lok_sabha.csv

```diff
@@ -28,20 +28,27 @@ ocd-division/country:in/state:ap/cd:visakhapatnam,Andhra Pradesh Lok Sabha const
 ocd-division/country:in/state:ap/cd:vizianagaram,Andhra Pradesh Lok Sabha constituency Vizianagaram,,
 ocd-division/country:in/state:ar/cd:arunachal_east,Arunachal Pradesh Lok Sabha constituency Arunachal East,,
 ocd-division/country:in/state:ar/cd:arunachal_west,Arunachal Pradesh Lok Sabha constituency Arunachal West,,
-ocd-division/country:in/state:as/cd:autonomous_district,Assam Lok Sabha constituency Autonomous District (ST),,
+ocd-division/country:in/state:as/cd:autonomous_district,Assam Lok Sabha constituency Autonomous District (ST),,2024
 ocd-division/country:in/state:as/cd:barpeta,Assam Lok Sabha constituency Barpeta,,
+ocd-division/country:in/state:as/cd:darrang_udalguri,Assam Lok Sabha constituency Darrang-Udalgiri,2024,
 ocd-division/country:in/state:as/cd:dhubri,Assam Lok Sabha constituency Dhubri,,
 ocd-division/country:in/state:as/cd:dibrugarh,Assam Lok Sabha constituency Dibrugarh,,
+ocd-division/country:in/state:as/cd:diphu,Assam Lok Sabha constituency Diphu (ST),2024,
 ocd-division/country:in/state:as/cd:gauhati,Assam Lok Sabha constituency Gauhati,,
+ocd-division/country:in/state:as/cd:guwahati,Assam Lok Sabha constituency Guwahati,,
 ocd-division/country:in/state:as/cd:jorhat,Assam Lok Sabha constituency Jorhat,,
-ocd-division/country:in/state:as/cd:kaliabor,Assam Lok Sabha constituency Kaliabor,,
-ocd-division/country:in/state:as/cd:karimganj,Assam Lok Sabha constituency Karimganj (SC),,
+ocd-division/country:in/state:as/cd:kaliabor,Assam Lok Sabha constituency Kaliabor,,2024
+ocd-division/country:in/state:as/cd:karimganj,Assam Lok Sabha constituency Karimganj,,
+ocd-division/country:in/state:as/cd:kaziranga,Assam Lok Sabha constituency Kaziranga,2024,
 ocd-division/country:in/state:as/cd:kokrajhar,Assam Lok Sabha constituency Kokrajhar (ST),,
 ocd-division/country:in/state:as/cd:lakhimpur,Assam Lok Sabha constituency Lakhimpur,,
-ocd-division/country:in/state:as/cd:mangaldoi,Assam Lok Sabha constituency Mangaldoi,,
+ocd-division/country:in/state:as/cd:mangaldoi,Assam Lok Sabha constituency Mangaldoi,,2024
+ocd-division/country:in/state:as/cd:nagaon,Assam Lok Sabha constituency Nagaon,,
 ocd-division/country:in/state:as/cd:nowgong,Assam Lok Sabha constituency Nowgong,,
 ocd-division/country:in/state:as/cd:silchar,Assam Lok Sabha constituency Silchar,,
-ocd-division/country:in/state:as/cd:tezpur,Assam Lok Sabha constituency Tezpur,,
+ocd-division/country:in/state:as/cd:silchar_sc,Assam Lok Sabha constituency Silchar (SC),,

```

Comment:
Hmm, I think we shouldn't be adding silchar_sc. We don't add the (SC) as part of the ID in other cases. (We shouldn't add the alias either, if we remove it here.)

Reply:
HI @jpmckinney  We have removed gauhati and nowgong from loksabha.csv & removed silchar_sc from both loksabha.csv and from alias.csv & have updated this changes in country-in.csv.